src/server.py

```diff
@@ -429,10 +406,7 @@ def run_seta_details_query():
         today = datetime.now()
         date = today.strftime("%Y-%m-%d")
 
-    db = create_db_connnection()
-    cursor = db.cursor()
-    query = "select jobtype from seta where date='%s 00:00:00'" % date
-    cursor.execute(query)
+    query = session.query(Seta.jobtype).filter(Seta.date == date).all()
```

Comment:
will this work for date?  I assume it will, but previously we had date='%s 00:00:00' to add the time stamp- is that needed?